tests/load/locustfiles/stored.py

```diff
@@ -184,23 +193,23 @@ def connect_and_unsubscribe(self):
         self.recv_message()
         self.close()
 
-    @task(weight=20)
+    @task(weight=30)
     def connect_and_read(self) -> None:
         """connect_and_hello then disconnect"""
+        if not self.initialized:
+            logger.debug("Task 'connect_and_read' skipped.")
+            return
         self.connect_and_hello()
         self.close()
 
-    def connect_and_register(self) -> None:
-        """Initialize the WebSocket and Hello/Register initial channels"""
+    def connect(self) -> None:
```

Comment:
Can we add a comment to this to explain what this is really doing? It took me a bit to work it out.